### PR TITLE
ci: job to update OwlBot's sha256

### PR DESCRIPTION
**Pull request by suztomo**
A job to periodically update OwlBot's sha256 value.

### CODE REVIEW

File: .github/workflows/owlbot_sha256.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.7'
+        architecture: 'x64'
+    - run: python3 -m pip install --require-hashes -r .github/requirements.txt
+    - run: python3 generate-readme.py
+    - name: Replace sha256 value in OwlBot to the latest value
+      shell: bash
+      run: |
+        sha256=$(gcloud container images describe gcr.io/cloud-devrel-public-resources/owlbot-java --format 'value(image_summary.digest)')
```
Comment (suztomo) [inline .github/workflows/owlbot_sha256.yaml:36]: Ubuntu latest should have gcloud command https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md

Comment (JoeWang1127) [reply .github/workflows/owlbot_sha256.yaml:33]: do you mean `Google Cloud SDK 421.0.0` in this [section](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#cli-tools)?

Comment (suztomo) [reply .github/workflows/owlbot_sha256.yaml:33]: Yes

File: .github/workflows/owlbot_sha256.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+on:
+  workflow_dispatch:
+  schedule:
+  # every hour
+  - cron: '0 * * * *'
```
[the remark this answers is not in the document]
Comment (suztomo) [reply .github/workflows/owlbot_sha256.yaml:19]: Daily, hourly, or every 4 hours, all should work.

**Review by JoeWang1127 — APPROVED** [review]

**Pull request merged**